content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md

```diff
@@ -19,11 +19,11 @@ With the Palette Nested Cluster option, you can deploy additional Kubernetes clu
 
 There are certain advantages to use a nested cluster environment. For example, you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
```

Comment:
With the Palette Nested Cluster option, you can deploy additional Kubernetes clusters, within an existing cluster, to provide teams with nested cluster environments.

There are certain advantages to use a nested cluster environment. For example, you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.

the above paragraph is information redundancy from the landing page overview. We can consider removing this @Siboney1

Reply:
Let's keep these paragraphs. It explains why you are using it with Palette in the quickstart and also provides a nice segue instead of jumping right in.

Reply:
This is needed. If they arrive at this page from an internal or external search, it will give them a brief synposis of what this page is about and how it relates to their find. User flow is not always linear.

Reply:
they can always refer the overview, from the subpage